src/olympia/devhub/templates/devhub/new-landing/components/navigation.html

```diff
@@ -1,25 +1,27 @@
-<nav class="DevHub-Navigation {% if user_authenticated %}DevHub-Navigation--signedin{% endif %}">
-  <ul>
-    <li>
-      <a href="#" class="Logo {% if user_authenticated %}Logo--secondary{% endif %}">
-        <span>{{ _('Add-on Developer Hub') }}</span>
-      </a>
-    </li>
+<nav class="DevHub-Navigation content {% if user_authenticated %}scheme-light{% else %}scheme-medium{% endif %}">
+  <div>
+    <ul>
+      <li>
+        <a href="#" class="Logo {% if user_authenticated %}Logo--secondary{% endif %}">
```

Comment:
ah yeah, I wanted to make a "content and links only" run later once the styling and general things are done so that I don't end up changing it back and forth all the time.